### ISSUE TITLE
Enable while paused or accessing inventory

### ISSUE THREAD

**Issue by NotSenser** [CIOCOLATA47/AutoWalk#5]
Is there any way to enable/keep the mod active while the game is paused or the user is accessing the inventory?

I currently make use of an Auto Clicker and an Auto Fishing Mod which both remain active while the pause and inventory screens are up and would like to know if the same functionality is at all achievable with this mod

**Comment by CIOCOLATA47** [comment]
Yeah you can open any gui the mod will still work and make you walk
